appengine/genetics/style.css

```diff
@@ -15,29 +15,343 @@
   position: relative;
 }
 
+#displayContent {
+  position: absolute;
+  width: 400px;
+  height: 400px;
+  overflow: hidden;
+}
+
 #display {
-  border: 1px solid #ddd;
-  vertical-align: bottom;
+  position: absolute;
+  width: inherit;
+  height: inherit;
+  background: #ccc;
+  z-index: -1;
+}
+
+.endGameDisplay {
+  position: absolute;
+  z-index: 1;
+  top: 25px;
+  left: 25px;
+  width: 350px;
+  height: 350px;
+  border-radius: 20px;
+  background: rgb(255, 255, 255);
+  background: rgba(255, 255, 255, 0.8);
```

Comment:
Google's CSS style guide recommends .8 rather than 0.8.  The opposite is recommended for JS.  Yay consistency!